### DISCUSSION TITLE
Experience Fragment enhancement to allow remote offer from Adobe Target

### DISCUSSION THREAD

**Discussion by epsilon-asutosh** [adobe/aem-core-wcm-components#1403] [Ideas]
If we want to load dynamic offer content from different experience fragment, can we enhance the component to allow the logic to replace the dynamic content based on the id provided in the component using the mobox script for at.js

In this way while authoring the XF component, author will need to select the default content from XF path and provide an id in the component. When the user is qualified for the segment, the respective segment content will be replaced on the webpage and user will see the updated content. If the user is not qualified for the segment, user will see the default content.

**Comment by vladbailescu** [comment]
@epsilon-asutosh , we would welcome a contribution for integrating Experience Fragments with Adobe Target, looking forward to your pull request.

**Reply by epsilon-asutosh** [reply]
@vladbailescu I was trying to commit my changes but it looks like I do not have previliages to commit my code. It gives me 403 response with the below error message.
Permission to adobe/aem-core-wcm-components.git denied to epsilon-asutosh.

**Reply by vladbailescu** [reply]
@epsilon-asutosh You will need to fork the repository, create a branch and commit the changes to your fork. Then you can send a pull request from that.

See also https://github.com/adobe/aem-core-wcm-components/blob/master/CONTRIBUTING.md#contributing-code-

**Reply by epsilon-asutosh** [reply]
@vladbailescu I have checked-in the code and here is the PR: https://github.com/adobe/aem-core-wcm-components/pull/1424
